libraries/lib-wave-track/WaveTrack.cpp

```diff
@@ -185,6 +185,54 @@ WaveTrack::Interval::Interval(const ChannelGroup &group,
 
 WaveTrack::Interval::~Interval() = default;
 
+void WaveTrack::Interval::SetName(const wxString& name)
+{
+   ForEachClip([&](auto& clip) { clip.SetName(name); });
```

Comment:
This and other lambdas can use empty capture `[]` and that might generate slightly more efficient code, because then they convert to pointer-to-function, and the `std::function` implementation might optimize that case avoiding some heap allocation.